examples/nextjs-14-permit/src/app/api/permissions/route.ts

```diff
@@ -0,0 +1,75 @@
+import { NextResponse } from "next/server";
+import arcjet, {
+  detectBot,
+  shield,
+  slidingWindow,
+} from "@arcjet/next";
+import { currentUser } from "@clerk/nextjs/server";
+import { Permit } from "permitio";
+
+const aj = arcjet({
+  // Get your site key from https://app.arcjet.com
+  // and set it as an environment variable rather than hard coding.
+  // See: https://nextjs.org/docs/app/building-your-application/configuring/environment-variables
+  key: process.env.ARCJET_KEY!,
+  // Define a global characteristic that we can use to identify users
+  characteristics: ["fingerprint"],
+  rules: [
+    // Shield detects suspicious behavior, such as SQL injection and cross-site
+    // scripting attacks. We want to ru nit on every request
+    shield({
+      mode: "LIVE", // will block requests. Use "DRY_RUN" to log only
+    }),
+    slidingWindow({
+      mode: "LIVE",
+      max: 2,
+      interval: 1,
+    }),
+    detectBot({
+      mode: "LIVE",
+      block: ["AUTOMATED"],
+    }),
+  ],
+});
```

Comment:
This should be a singleton SDK since you use an client in more than 1 file. You might not be able to default all your rules then and you'll need to use `withRule` to add them before each handler.